libs/wingsdk/src/target-sim/platform.ts

```diff
@@ -11,4 +12,32 @@ export class Platform implements IPlatform {
   public newApp(appProps: any): any {
     return new App(appProps);
   }
+
+  public async storeSecrets(secrets: Record<string, string>): Promise<void> {
+    let existingSecretsContent = "";
+    try {
+      existingSecretsContent = fs.readFileSync("./.env", "utf8");
+    } catch (error) {}
+
+    const existingSecrets = existingSecretsContent
+      .split("\n")
+      .filter((line) => line.trim() !== "")
+      .reduce((s, line) => {
+        const [key, value] = line.split("=", 2);
+        s[key] = value;
+        return s;
+      }, {} as { [key: string]: string });
+
+    for (const key in secrets) {
+      existingSecrets[key] = secrets[key];
+    }
+
+    const updatedContent = Object.entries(existingSecrets)
+      .map(([key, value]) => `${key}=${value}`)
+      .join("\n");
+
+    fs.writeFileSync("./.env", updatedContent);
```

Comment:
Is it possible to ensure the loaded data stays in memory? Does secret.inflight.ts have access to the envs loaded in the simulator so that we don't need to deal with files?

Reply:
Yea you are right this can all stay in memory